package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "generator-loopback",
-  "version": "2.2.1",
```

Comment:
Please use `3.0.0-alpha.1`, to prevent our internal npm registry from containing a `3.0.0` version uploaded by our CI.